libs/protocol-client/src/lib/client/guild_user.ts

```diff
@@ -30,4 +31,9 @@ export class GuildUser extends BaseClient {
     });
     return updatedGuildUser.updateGuildUserCustom;
   }
+
+  public async createMany(args: CreateManyGuildUserMutationVariables) {
```

Comment:
For our methods we use bulk instead of many. Should probably be bulkCreate instead.